cabot/metricsapp/forms/grafana_elastic.py

```diff
@@ -20,6 +20,7 @@ class Meta:
             'time_range',
             'retries',
             'frequency',
+            'use_activity_counter',
```

Comment:
Please add [here](https://github.com/Affirm/cabot/blob/master/cabot/metricsapp/forms/elastic.py#L27) also (there's probably a way to make these configs simpler, sorry)

Reply:
done